cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -1993,30 +2010,18 @@ def test_obsm_values_no_X_embedding__non_spatial_dataset(self, validator_with_ad
             "WARNING: Validation of raw layer was not performed due to current errors, try again after fixing current errors.",
         ]
 
-    @pytest.mark.parametrize(
-        "assay_ontology_term_id, adata_spatial",
-        [
-            ("EFO:0010961", examples.adata_visium),
-            ("EFO:0030062", examples.adata_slide_seqv2),
-        ],
-    )
-    def test_obsm_values_no_X_embedding__spatial_dataset(
-        self, validator_with_adata, assay_ontology_term_id, adata_spatial
-    ):
-        validator = validator_with_adata
-        validator.adata.obsm["harmony"] = validator.adata.obsm["X_umap"]
-        validator.adata.uns = adata_spatial.uns
-        validator.adata.uns["default_embedding"] = "harmony"
-        validator.adata.obsm["spatial"] = validator.adata.obsm["X_umap"]
+    def test_obsm_values_no_X_embedding__visium_dataset(self, validator_with_visium_assay):
+        validator = validator_with_visium_assay
+        validator.adata.uns["default_embedding"] = "spatial"
```

Comment:
I am not familiar with the complete, required shape for `adata` here but I see there are a couple of differences between the previous version and `validator_with_visium_assay` (e.g. `validator.adata.obsm["harmony"]` is no longer defined, `validator.adata.uns["default_embedding"]` is now `spatial`). Are these updates expected/OK?

Reply:
yes, its expected

for non-spatial datasets, not having an X-prefix embedding like `X_umap` should result in an error (see `test_obsm_values_no_X_embedding__non_spatial_dataset`). 

However, for spatial datasets, we do not require an X-prefix embedding, just 'spatial' embedding. This test case was originally modeled after `test_obsm_values_no_X_embedding__non_spatial_dataset`, which used 'harmony' as its non-X-prefix embedding to trigger the error. But I realized spatial is already a non-X-prefix embedding so we didn't need to add another one to test.